### PR TITLE
Release Candidate v2.1.0

### PR DESCRIPTION
**Pull request by kirre-bylund**

### CODE REVIEW

File: Runtime/Game/LootLockerSDKManager.cs [hunk 1]
```diff
@@ -2403,327 +2348,440 @@ public static void RemoveAssetFromHeroLoadout(string assetID, string heroID, Act
...
         }
 
         /// <summary>
-        /// Get the loadout for the players default character.
+        /// Get the loadout for the players default class.
         /// </summary>
-        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerGetCurrentLoadouttoDefaultCharacterResponse</param>
-        public static void GetCurrentLoadOutToDefaultCharacter(Action<LootLockerGetCurrentLoadouttoDefaultCharacterResponse> onComplete)
+        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerGetCurrentLoadoutToDefaultClassResponse</param>
+        public static void voidGetCurrentLoadoutToDefaultClass(Action<LootLockerGetCurrentLoadoutToDefaultClassResponse> onComplete)
```
Comment (JohannesLoot) [inline Runtime/Game/LootLockerSDKManager.cs:2714]: Remove void from function name

File: Runtime/Game/Requests/PurchaseRequest.cs [hunk 2]
```diff
@@ -138,7 +143,7 @@ public static void AndroidPurchaseVerification(LootLockerAndroidPurchaseVerifica
             LootLockerServerRequest.CallAPI(endPoint.endPoint, endPoint.httpMethod, json, (serverResponse) => { LootLockerResponse.Deserialize(onComplete, serverResponse); });
         }
 
-        public static void PollOrderStatus(LootLockerGetRequest lootLockerGetRequest, Action<LootLockerCharacterLoadoutResponse> onComplete)
+        public static void PollOrderStatus(LootLockerGetRequest lootLockerGetRequest, Action<LootLockerPurchaseOrderStatus> onComplete)
```
Comment (JohannesLoot) [inline Runtime/Game/Requests/PurchaseRequest.cs:146]: Should we instead mark this as deprecated?
Noone is probably using it, but just to be consistent.

**Review by JohannesLoot — CHANGES_REQUESTED** [review]
Minor spelling-error on function (see comment), otherwise all good!
👍

File: Runtime/Game/LootLockerSDKManager.cs (hunk 1 above)
Comment (kirre-bylund) [reply Runtime/Game/LootLockerSDKManager.cs]: Done

File: Runtime/Game/Requests/PurchaseRequest.cs (hunk 2 above)
Comment (kirre-bylund) [reply Runtime/Game/Requests/PurchaseRequest.cs:146]: Done

**Review by AlmightyMikkel — APPROVED** [review]
🚋 Release train on the move 🚿

**Pull request merged**